src/dialog-user/components/dialog-user/dialogUser.ts

```diff
@@ -79,19 +83,24 @@ export class DialogUserController extends DialogClass implements IDialogs {
     };
     this.onUpdate({ data: outputData });
   }
+
   public validateFields() {
     const validations = {
       isValid: true,
-      messages: []
+      messages: [], // currently unused, should be?
```

Comment:
Hmm, maybe we need to standardize on how the HTML expects to use this, then. I could've sworn we had a place in the HTML that showed the overall messages but I guess not. It makes sense (to me, at least) to have individual validation error messages (something like "incorrect regex matching") versus overall validations like "Fields are being refreshed".

I think that's out of the scope of this PR though.